.github/workflows/install/action.yml

```diff
@@ -3,10 +3,12 @@ description: 'Set up and install dependencies'
 
 inputs:
   fetch-depth:
+    description: 'Number of commits to fetch'
     default: '1'
     required: false
   persist-credentials:
-    default: true
+    description: 'Whether to persist credentials'
+    default: "true"
```

Comment:
Why this needs to be string?

Reply:
<img width="1278" alt="screen-shot" src="https://user-images.githubusercontent.com/15878786/215061185-1103f206-762b-42a8-ba46-538bd7065151.png">